.gitignore

```
@@ -190,4 +190,7 @@ cython_debug/
 # vendor/
 
 # iDRAC credentials
-idrac_vars.yaml
\ No newline at end of file
+idrac_vars.yaml
+
+# Personal containerfiles used for local testing
+**/tmp.Containerfile
```

```suggestion
tmp.Containerfile
```

no need for the `**` right?